openmdao/drivers/scipy_optimizer.py

```diff
@@ -319,6 +352,12 @@ def run(self):
         else:
             jac = None
 
+        if opt in _hessian_optimizers:
+            from scipy.optimize import BFGS  # FIXME temporary, should be option. (P.O.)
```

Comment:
The #FIXME here is related to your question on hessian computation, right? So this is really a TODO in the future if we supported Hessians?

Reply:
indeed, it should be a TODO